src/World.h

```diff
@@ -188,7 +188,13 @@ class cWorld  // tolua_export
 	virtual void BroadcastChatSuccess(const AString & a_Message, const cClientHandle * a_Exclude = nullptr) override { BroadcastChat(a_Message, a_Exclude, mtSuccess); }
 	virtual void BroadcastChatWarning(const AString & a_Message, const cClientHandle * a_Exclude = nullptr) override { BroadcastChat(a_Message, a_Exclude, mtWarning); }
 	virtual void BroadcastChatFatal  (const AString & a_Message, const cClientHandle * a_Exclude = nullptr) override { BroadcastChat(a_Message, a_Exclude, mtFailure); }
-	virtual void BroadcastChatDeath  (const AString & a_Message, const cClientHandle * a_Exclude = nullptr) override { BroadcastChat(a_Message, a_Exclude, mtDeath); }
+	virtual void BroadcastChatDeath  (const AString & a_Message, const cClientHandle * a_Exclude = nullptr) override
+	{
+		if (ShouldBroadcastDeathMessages())
+		{
+			BroadcastChat(a_Message, a_Exclude, mtDeath);
```

Comment:
Can't you do this check in `cWorld::BrodcastChat`?
This will make the diff smaller

Reply:
Yeah, good idea!